.github/workflows/gamemaker_build.yml

```
@@ -6,29 +6,21 @@ on:
         default: false
-      development:
-        required: false
-        type: boolean
-        default: true
-    outputs:
-      suffix:
-        value: ${{ jobs.build.outputs.suffix }}
+      build_date:
+        required: true
+        type: string
+        default: "unknown"
```

actionlint (1.7.4)

input "build_date" of workflow_call event has the default value "unknown", but it is also required. if an input is marked as required, its default value will never be used

(events)